From a3a9e63a2348ca99e862167e93ebf3e0edf7ff11 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Tue, 13 Mar 2012 16:57:48 +0800
Subject: [PATCH 1162/2463] ENGR00176649-1 sabrelite_SD ARD pfuze: update
 pfuze driver for pfuze100 1.0

Since some power rail is differnet between pfuze100 0.1 and pfuze100 1.0,
remove PFUZE100_FIRST_VERSION and change PFUZE100_SW4_VOL6 define.

Signed-off-by: Robin Gong <b38343@freescale.com>
---
 include/linux/mfd/pfuze.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/linux/mfd/pfuze.h b/include/linux/mfd/pfuze.h
index 38c4e24..92e34af 100644
--- a/include/linux/mfd/pfuze.h
+++ b/include/linux/mfd/pfuze.h
@@ -87,8 +87,8 @@ unsigned int pfuze_get_flags(struct mc_pfuze *mc_pfuze);
 #define PFUZE100_VGEN6		15
 #define PFUZE100_SW2_VOL6	1
 #define PFUZE100_SW3_VOL6	0
-#define PFUZE100_SW4_VOL6	0
-#define PFUZE100_FIRST_VERSION
+#define PFUZE100_SW4_VOL6	1
+/*#define PFUZE100_FIRST_VERSION*/
 
 struct regulator_init_data;
 struct pfuze_regulator_init_data {
-- 
1.7.7.4

